src/char/hammerdin.py

```diff
@@ -1,3 +1,4 @@
+from cgitb import text
```

Comment:
where is this supposed to be used?

Reply:
could be fragments from previous versions - happy to have it removed if it does not break the script.

Reply:
This was a change from whatever's on master. So you'd have to tell me if it breaks or not.